docs/src/pages/guides/understand-mui-packages/understand-mui-packages.md

```diff
@@ -34,35 +34,42 @@ For this reason, abstracting into smaller packages not only allows MUI to grow o
 
 The packages can be categorized into 3 layers, as shown in the picture below:
 
-<img src="/static/images/packages/mui-packages.png" style="width: 700px; max-width: 100%;" />
+<img src="/static/images/packages/mui-packages.png" style="display: block; width: 448px; max-width: 100%; margin: auto;" />
```

Comment:
remove spacing in the image to make it looks good in mobile.

Reply:
This style seems to already comes by default

```suggestion
<img src="/static/images/packages/mui-packages.png" style="display: block; width: 448px; margin: auto;" />
```